src/oas2/__tests__/operation.test.ts

```diff
@@ -90,4 +91,52 @@ describe('transformOas2Operation', () => {
       }),
     ).toHaveProperty('deprecated', true);
   });
+
+  test('given malformed parameters should translate operation with those parameters', () => {
+    const document: Partial<OpenAPIObject> = {
+      swagger: '2.0',
+      paths: {
+        '/users/{userId}': {
+          get: {
+            parameters: [
+              {
+                in: 'header',
+                name: 'name',
+                schema: {
+                  type: 'string',
+                },
```

Comment:
OpenAPI v2 parameters don't have a "schema" property if you want to be technically correct 🤓 

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameters-definitions-object

```ts
type: 'string',
```